lib/rubocop/cop/rails/find_each.rb

```diff
@@ -15,7 +15,9 @@ module Rails
       class FindEach < Cop
         MSG = 'Use `find_each` instead of `each`.'.freeze
 
-        SCOPE_METHODS = %i[all where not].freeze
+        SCOPE_METHODS = %i[
+          all includes joins left_outer_joins not unscoped where
```

Comment:
If `includes`, `joins` and `left_outer_joins` were added, I thought that we could also add `eager_load`, `left_joins`, `preload` and `references`.

And this will also change the behavior, so it should be written in the `Changes` section of CHANGELOG 😃 
https://github.com/rubocop-hq/rubocop/blob/63cdaa28a32a7ab475d61666564af468e61078e3/CHANGELOG.md#changes

Reply:
Makes sense to me. I'll update the PR. Thanks